controllers/api_v1/keyword_test.go

```diff
@@ -105,6 +105,135 @@ func TestKeywordAPIControllerDbTestSuite(t *testing.T) {
 	suite.Run(t, new(KeywordAPIControllerDbTestSuite))
 }
 
+func (s *KeywordAPIControllerDbTestSuite) TestFetchKeywordsWithValidParams() {
+	keyword := models.Keyword{UserID: s.user.ID, Keyword: faker.Name()}
+	db.GetDB().Create(&keyword)
+
+	headers := http.Header{}
+	headers.Set("Authorization", "Bearer test-access")
+
+	resp := testHttp.PerformRequest(s.engine, "GET", "/api/v1/keywords", headers, nil)
+	respBodyData, _ := ioutil.ReadAll(resp.Body)
+	var parsedRespBody map[string][]api_helper.DataResponseObject
+	_ = json.Unmarshal(respBodyData, &parsedRespBody)
```

Comment:
@gutakk Regarding [our RFC](https://github.com/nimblehq/compass/issues/469) we should handle all errors on the test files also 🙌🏼

Reply:
Fixed in e0deb9969b647a59116159ce3928532aa5ac3f8c